app/assets/stylesheets/govuk_publishing_components/components/_related-navigation.scss

```diff
@@ -42,11 +42,7 @@
 .gem-c-related-navigation__link {
   list-style-type: none;
   margin-top: govuk-spacing(3);
-  @include govuk-font($size: 16, $weight: regular, $line-height: 1.5);
-
-  @include govuk-media-query($from: tablet) {
-    margin-top: 5px;
-  }
+  @include govuk-font($size: 16, $weight: regular, $line-height: 1.28);
```

Comment:
On mobile the line height looks too small, so the bottom of the top line gets cropped a little - note the `p`s aren't fully there.

<img width="270" alt="Screen Shot 2019-08-13 at 15 30 05" src="https://user-images.githubusercontent.com/861310/62950123-5d2b8280-bddf-11e9-95e4-bfe7aa8a6ec9.png">

Reply:
Which browser?

Reply:
Fixed - good spot, thanks!